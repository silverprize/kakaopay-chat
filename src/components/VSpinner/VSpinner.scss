@import '../../scss/variables';

@keyframes spinner {
  to { transform: rotate(360deg); }
}

.spinner {
  display: inline-block;
  width: 2.5rem;
  height: 2.5rem;
  border: 0.3rem solid $yellow-color;
  border-right-color: transparent !important;
  border-radius: 50%;
  animation: spinner .75s linear infinite;

  &_color_light {
    border-color: $light-color;
  }
}
