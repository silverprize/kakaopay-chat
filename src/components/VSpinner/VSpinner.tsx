import { Component, Vue } from 'vue-property-decorator'
import './VSpinner.scss'

@Component
export default class VSpinner extends Vue {
  render() {
    return <div class="spinner" />
  }
}
