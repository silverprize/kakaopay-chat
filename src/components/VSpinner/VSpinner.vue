<template functional>
  <div class="spinner" />
</template>

<script lang="ts">
import './VSpinner.scss'
import { Component, Vue } from 'vue-property-decorator'

@Component
export default class VSpinner extends Vue {}
</script>
