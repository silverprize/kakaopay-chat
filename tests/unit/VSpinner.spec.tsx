import VSpinner from '@/components/VSpinner/VSpinner'
import { mount } from '@vue/test-utils'

describe('VSpinner', () => {
  test('렌더링 테스트', () => {
    const wrapper = mount(VSpinner)
    expect(wrapper).toMatchSnapshot()
  })
})
