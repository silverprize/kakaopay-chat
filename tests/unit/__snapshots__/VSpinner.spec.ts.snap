// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VSpinner 렌더링 1`] = `<div class="spinner"></div>`;
